packages/wasm-edit/src/index.js

```
@@ -31,7 +31,7 @@ function sortBySectionOrder(nodes: Array<Node>) {
     }
 
     // $FlowIgnore ensured above
```

Can the FlowIgnore can be removed now? Did it hide the error?

---

Yes indeed, thanks.